OpenEnergyDashboard/OED
# Bugfix for loss of map graph display when in mobile

# Description
This bug fix removes the styling `alignItems: 'center'` from the `div` wrapper around the `PlotNavComponent`, which displays  the graphs in the application. With this change, the entire map should be available via scrolling.

Explanation for bug:
The styling did not affect other maps as they were drawn using `Plot` while considering the viewport's width and height in terms of percentages (100% of width and height). The styling caused this bug for map graphs because an image is used (which is not adjusted to the viewport's height and width), and the styling caused the "start" of the map to be at the center.

Fixes #1347 

## Type of change
N/A
- [ ] Note merging this changes the database configuration.

## Checklist
- [x] I have followed the [OED pull request](https://openenergydashboard.org/developer/pr/) ideas
- [N/A] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.org/developer/cla/) and each author is listed in the Description section.

## Limitations
No limitations

## Manual Tests Done
I manually tested the app's behavior pre and post fix for both desktop and mobile viewport, considering:
1.  Line graphs still display properly.
2. Bar graphs still display properly.
3. Compare graphs still display properly.
4. Radar graphs still display properly.
5. Map graphs now display properly.
6. Navigating between graphs and the Menu appears to be working properly.

Note: Happy to provide screenshots of all other maps still working properly, but will only upload the working map view to start with unless more is requested.

https://github.com/user-attachments/assets/4150c567-43bd-49ed-a599-c827083b597d
